src/Components/Books/Book.css

```diff
@@ -1,3 +1,121 @@
-* {
-  margin: 0;
+.bookCard {
+  width: 90%;
+  height: 10.625rem;
+  padding: 20px;
+  margin: auto;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  border: 1px solid #f0ecec;
+  margin-bottom: 10px;
+}
+
+.bookCard .book_container_left {
+  width: 45%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.category {
+  font-size: 1.2rem;
+  color: #868686;
+}
+
+.book_container_left p.title {
+  font-size: 1.4rem;
+  font-weight: bold;
+  font-family: 'Times New Roman', Times, serif;
```

Comment:
- [ ] Great work styling this application. The `font-families` used in the design template are `Roboto` and `Monstserrat`. Kindly apply this change. 👍